INSTALL.rst

```diff
@@ -257,6 +257,43 @@ location specified in ``haas.cfg``) and initialize its tables::
 
   haas init_db
 
+
+Authentication and Authorization
+--------------------------------
+
+HaaS includes a pluggable architecture for authentication and authorization.
+HaaS ships with two authentication backends. One uses HTTP basic auth, with
+usernames and passwords stored in the haas database. The other is a "null"
+backend, which does no authentication or authorization checks. This can be
+useful for testing and experimentation but *should not* be used in production.
+You must enable exactly one auth backend.
+
+Database Backend
+^^^^^^^^^^^^^^^^
+
+To enable the database backend, make sure the **[extensions]** section of
+``haas.cfg`` contains::
+
+  haas.ext.auth.database =
+
+Then initialize the database as described above. You will need to add an
```

Comment:
No, that's correct. the syntax we use for extension in general is a little weird, but it's how we've been doing things. The presence of the option causes it to be loaded; it's value isn't used. This is in line with existing practice.

Re; "above," there's a section earlier in the document about initializing the database. It's actually a bit awkwardly arranged as is. I improve this somewhat in #556, which I just opened.